mod_app/templates/analysis_list.html

```diff
@@ -12,7 +12,9 @@ <h3>
     <div class="card-area">
         {% for analysis in analysis_list %}
             <div class="card card--analysis">
-                <h4 class="card__heading"><a href="{% url 'analysis_detail' pk=analysis.pk %}">{{analysis.title}}</a></h4>
+                <h4 class="card__heading"><a
+                    href="{% url 'analysis_detail' pk=analysis.pk %}"
+                >{{analysis.title}}</a></h4>
                 <div class="card__info">
                     <dl>
                         <dt>{{analysis.content|safe|truncatewords:50}} </dt>
```

Comment:
The width of dt elements appears to be constrained in normalise.css, which makes the text only take up about 2/3 of the width of the card. Is that what you want?

Reply:
Potentially not but I'm also not sure they'll want to keep the cards; they may prefer a table for these as well. I'll leave it for now but make a note